cms/templates/product_page.html

```diff
@@ -36,6 +36,9 @@
   <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/hls.js@latest"></script>
   <script src="{% static 'js/course_detail.js' %}"></script>
   <script src="{% static 'js/hero.js' %}"></script>
+  <script src="{% static 'js/testimonials_carousel.js' %}"></script>
+  <script src="{% static 'js/courseware_carousel.js' %}"></script>
+  <script src="{% static 'js/faculty_carousel.js' %}"></script>
 {% endblock %}
```

Comment:
@ahmed-arbisoft a question here, I remember we have companies slider as well. Can we make it separate as well ?

Reply:
Merged already, will do so in my next PR, whichever ticket it will be for